audformat/core/database.py

```diff
@@ -289,6 +290,47 @@ def drop_tables(
         for table_id in table_ids:
             self.tables.pop(table_id)
 
+    def files_duration(
+            self,
+            files: typing.Union[str, typing.Sequence[str]],
+    ) -> typing.Union[pd.Timedelta, pd.Series]:
+        r"""Duration of file(s) in the database.
+
+        Note that durations are cached,
+        i.e. changing the files on disk after calling
+        this function can lead to wrong results.
+        The cache is cleared when the
+        database is reloaded from disk.
+
+        Use ``db.files_duration(db.files).sum()``
+        to get the total duration of a database.
+        Or ``db.files_duration(db['table'].files).sum()``
+        to get the total duration of a table.
```

Comment:
It's not so nice that we now have two functions that can do the same thing. E.g. `db.files_duration(db.files).sum()`
and `audformat.utils.duration(db.files)`. As I don't have to add the `.sum()` at the end, I think I would pick the later as a user as I will not be aware that it cannot use the caching.

Reply:
The advantage I see is that you can easily apply other operations, e.g. `db.files_duration(db.files).max()` or  `db.files_duration(db.files).mean()`.

Reply:
Yes, I also see the advantage. I'm just worried that most of the users will simply continue using `audformat.utils.duration()`, but I think we simply have to life with it. It's also still great to have it as it provides the possibility to calculate duration for data frames that are not part of any database.

Reply:
I think it is not obvious what "the total duration of a table" means for a segmented table.
Maybe we should change to
```suggestion
        to get the total duration of all files assigned to a table.
```

Reply:
Changed to:

![image](https://user-images.githubusercontent.com/10383417/140722180-63ee090f-779a-4776-a9cd-4408937bbf5c.png)